mso/provider.go

```diff
@@ -58,6 +59,12 @@ func Provider() terraform.ResourceProvider {
 					"nd",
 				}, false),
 			},
+			"retries": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("MSO_RETRIES", nil),
+				Description: "Number of retries for REST API calls.",
```

Comment:
Shall we change this to? ->
Description: "Number of retries for REST API calls. Defaults to `2`.",